### ISSUE TITLE
Add Sengi

### ISSUE THREAD

**Issue by philclifford** [wimpysworld/deb-get#803]
## Requesting a new app ✨

If you are requesting a new app to be added to `deb-get` please complete the
following section, only one app per issue please.

 - [X] I have checked that this app does not already have a pending request and has not previously been declined.
 - App Name: Sengi
 - Download URL:  https://github.com/NicolasConstant/sengi/releases/download/1.2.0/Sengi-1.2.0-linux.deb
 - Website URL: https://nicolasconstant.github.io/sengi/
 - App Summary: A Mastodon and Pleroma desktop focused client.

**Comment by philclifford** [comment]
Nope - still misbehaving about versions

**Comment by philclifford** [comment]
In case anyone wants to work around this (not uncommon) problem, if you add a line into the definition with 

```shell
VERSION_PUBLISHED=${VERSION_PUBLISHED/<github release installed>/<dpkg installed version>}
```
then when a new release arrives it will be installed, but you'll need to edit the above line (so own repo or 99-local.d override behaviour until some genius writes a self-maintaining self-patching function ...)

I have this in `99-local.d`

```shell
DEFVER=1
get_github_releases "NicolasConstant/sengi" "latest"
if [ "${ACTION}" != prettylist ]; then
    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d '"' -f4)"
    VERSION_PUBLISHED="$(echo "${URL}" | cut -d '-' -f2)"
fi
VERSION_PUBLISHED=${VERSION_PUBLISHED/1.2.0/1.1.4-1087}
PRETTY_NAME="Sengi"
WEBSITE="https://nicolasconstant.github.io/sengi/"
SUMMARY="A Mastodon and Pleroma desktop focused client."
```

This gives scary but ignorable warnings because the app is currently marked as deprecated because of this behaviour, but otherwise produces 

```shell
  [*] WARNING! Deprecated package sengi detected. It will no longer receive updates, and keeping it installed is considered unsafe.
  [!] Please remove it with: deb-get purge sengi
Sengi
  Package:	sengi
  Repository:	99-local
  Updater:	deb-get
  Installed:	1.1.4-1087
  Published:	1.1.4-1087
  Architecture:	amd64
  Download:	https://github.com/NicolasConstant/sengi/releases/download/1.2.0/Sengi-1.2.0-linux.deb
  Website:	https://nicolasconstant.github.io/sengi/
  Summary:	A Mastodon and Pleroma desktop focused client.
```
